arithmetization/src/main/java/net/consensys/linea/zktracer/types/MemoryRange.java

```diff
@@ -113,6 +113,6 @@ public boolean isEmpty() {
   }
 
   public MemoryRange snapshot() {
-    return new MemoryRange(this.contextNumber, this.range.snapshot(), this.rawData);
+    return new MemoryRange(contextNumber, range.snapshot(), rawData.copy());
```

Comment:
not sure it's usefull as rawData is final ... better to just store a pointer instead of a copy, wdyt, do we have any chance we may require a copy @OlivierBBB  ?

Reply:
The `rawData` being final only means that the reference is set in stone. The contents not so much. I'm for this change. We will in any case see if this creates issues. Let's do the safe thing for now.